spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -1199,3 +1199,108 @@ The following metrics are produced for each channel declared by the application
 // ==== Custom message types
 
 // Implementations may provide custom subtypes of `Message` as an end user API, for the purpose of providing additional metadata associated with the messaging provider. If an application declares that it consumes or produces a subtype of the `Message` wrapper, the implementation should validate that the messaging provider supports that message wrapper type, and raise a deployment exception if it doesn't.
+
+== Use Reactive Messaging in the imperative world
```

Comment:
Not sure I agree that this is what we're talking about.

To me, it's more of a "bridge" allowing a developer to receive data from an imperative endpoint, and convert it to a reactive message

Reply:
Bridging Reactive and Imperative? Feel free to suggest a better heading.

Reply:
Maybe "Constructing messages from non-reactive code"?

I consider the content of these paragraphs to discuss topics that are irrelevant to what is being done. For instance, the sentence on bridging implies you can mix imperative and reactive programming in a single place, which isn't the case. What is defined is a way to expose messages from imperative code into a reactive stream. It's not a general solution where you can have `@Outgoing` on a JAX-RS Resource method, etc

Reply:
This feature enables jax-rs resources to publish messages to a topic. JAX-RS is imperative and it is able to connect to reactive world.

Reply:
I will rephrase some texts. I am not sure about "Constructing messagings from non-reactive code". How about "Sending messages from imperative code to a reactive flow? Any better suggestions welcome?

Reply:
reworded